Header header
float32[] raw_data
float32[] gaussians
float32[] features
uint32[] split_inds1
uint32[] split_inds2
uint32[] split_inds3
uint32 traj_id
uint32 label
float32 detect_delay
